addon/utils/tools.js

```diff
@@ -39,13 +39,14 @@ export default {
     name: 'Polygon',
     title: 'Polygon Tool',
     options: [
-      { name: 'Fill Color', type: 'color', id: 'style.fillColor' },
+      { name: 'Fill Color', type: 'color', id: 'style.fillColor', isCheckbox: true },
```

Comment:
Can we call `isCheckbox` something more context based, like `fillOptional` or `canToggleFillOpacity`